src/api/object.js

```diff
@@ -5,6 +5,7 @@ const DAGLink = require('ipfs-merkle-dag').DAGLink
 const promisify = require('promisify-es6')
 const bs58 = require('bs58')
 const bl = require('bl')
+const cleanMultihash = require('../clean-multihash')
 
 module.exports = (send) => {
   const api = {
```

Comment:
Another option would be to use default values for arguments (which was implemented in node.js v6), but I guess we don't do full ES6 in our code base?